pkg/metadata/inventories/inventories.go

```diff
@@ -124,48 +132,48 @@ func createCheckInstanceMetadata(checkID, configProvider string) *CheckInstanceM
 
 // CreatePayload fills and returns the inventory metadata payload
 func CreatePayload(hostname string, ac AutoConfigInterface, coll CollectorInterface) *Payload {
-	checkCacheMutex.Lock()
-	defer checkCacheMutex.Unlock()
+	checkMetadataMutex.Lock()
+	defer checkMetadataMutex.Unlock()
 
-	checkMetadata := make(CheckMetadata)
+	checkMeta := make(CheckMetadata)
```

Comment:
I think the new similar naming of the variables does make this a bit harder to disambiguate during reading and may be more likely to cause a programmer mistake (`checkMeta` + `checkMetadata` in the new version vs `checkMetadata` + `checkMetadataCache` in the old one).

Reply:
Well, both are pretty confusing -- that's a good point.

Reply:
I also found `checkMeta` to be confusing, more that `checkMetadata`+ `checkMetadataCache`.